exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include <iomanip>
+#include <algorithm>
+
+
+
+std::string getWord(std::ifstream& stream) {
+  /* Function accept an input stream object and return a string from
+  input stream by reading stream 1 character at a time  and formats it  
+  by making it lowercase and removing punctuation marks then 
+  concatenates it to a string until a space ,newline or '-'  character.
+  */
+  std::string outputString{};
```

Comment:
This comment doesn't relate to anything new you've changed in this 2nd round of commits but just thought of it now.
In general in the Mantid codebase we don't tend to put the empty braces around the declaration of a string because "std::string outputString" has the same effect ie default constructor is called and you end up with an empty string. This isn't explicity documented in our coding standard but perhaps comes under the general approach of matching the style of the code that's already there. 
So please can you remove the braces from this and any other string declarations?